### PR TITLE
GifMe Command

### PR DESCRIPTION
**Pull request by mp7786**
This command is intended to return a random GIF when called.

### CODE REVIEW

File: bot/bot_secrets.py [hunk 1]
```diff
@@ -108,6 +109,18 @@ def bot_prefix(self, value: str) -> None:
         if self._botPrefix:
             raise ConfigAccessError(f'bot_prefix has already been initialized')
         self._botPrefix = value
+    
+    @property
+    def gif_me(self) -> str:
```
Comment (Jay-Madden) [inline bot/bot_secrets.py:114]: we should postfix this value with token so the json and the property should be gifMeToken

Comment (mp7786) [reply bot/bot_secrets.py:114]: This would be in the bot secrets template?

Comment (Jay-Madden) [reply bot/bot_secrets.py:114]: ya, and in the BotSecrets class, just add Token to the end of everything

File: bot/bot_secrets.py [hunk 2]
```diff
@@ -131,4 +131,4 @@ def load_secrets(self, lines: str) -> None:
         self.bot_token = secrets['BotToken']
         self.database_name = secrets['DatabaseName'] or 'ClemBot'
         self.bot_prefix = secrets['BotPrefix'] or '!'
-        self.gif_me = secrets["GifMe"]
+        self.gif_me_token = secrets["GifMe"]
```
Comment (Jay-Madden) [inline bot/bot_secrets.py:134]: you forgot to change the key to "GifMeToken"

File: BotSecrets.json.template [hunk 3]
```diff
@@ -3,5 +3,6 @@
     "ClientSecret": "",
     "BotToken": "",
     "DatabaseName": "",
-    "BotPrefix": ""
+    "BotPrefix": "",
+    "gifMeToken": ""
```
Comment (Jay-Madden) [inline BotSecrets.json.template:7]: This should also be capitalized in PascalCase like the other json attributes

**Pull request merged**